example-app/src/main/java/com/adyen/checkout/example/data/api/model/PaymentsRequest.kt

```diff
@@ -32,5 +32,6 @@ data class PaymentsRequestData(
     val reference: String,
     val channel: String,
     val lineItems: List<Item>,
+    val shopperEmail: String? = null,
```

Comment:
Can you test this with sessions? Maybe we need to add this field to the `SessionRequest` as well

Reply:
yes it's not working with session I'll add it in `SessionRequest`